internal/core/command/device_test.go

```diff
@@ -53,56 +55,65 @@ func TestExecuteGETCommandByDeviceIdAndCommandId(t *testing.T) {
 		deviceId    string
 		commandId   string
 		expectedErr error
+		request     *http.Request
```

Comment:
Depending on if we add the header functionality for GET commands, we should either remove this test argument since it is always nil, or pass in an actual request and assert that the `HttpCaller` was invoked with a request with the correct headers.